habitat_baselines/utils/common.py

```diff
@@ -345,3 +352,78 @@ def overwrite_gym_box_shape(box: Box, shape) -> Box:
     low = box.low if np.isscalar(box.low) else np.min(box.low)
     high = box.high if np.isscalar(box.high) else np.max(box.high)
     return Box(low=low, high=high, shape=shape, dtype=box.dtype)
+
+
+def get_scene_episode_dict(episodes: List[Episode]) -> Dict:
```

Comment:
Usually episodes are coming from dataset. This function can be replaces by `scene_ids` property:
https://github.com/facebookresearch/habitat-lab/blob/422a60fb025e9bcaf80bc1b6cf24d48d26e10b41/habitat/core/dataset.py#L131-L134

Reply:
The `scene_ids` property can help me get unique scene ids, but `get_scene_episode_dict` provides the scene-episode mapping. 

```
{'scene1': [{episode_1},{episode_2}], 'scene2': [{episode_3},{episode_4}]}
```

Maybe we can add this function to the dataset too?

Reply:
@mukulkhanna Absolutely if you think it's valuable.